components/ProfileInfo/ProfileInfo.module.css

```diff
@@ -0,0 +1,23 @@
+.box {
+    display: flex;
+}
+
+.fields {
```

Comment:
Hmm I don't know why this CSS rule got duplicated here since I think you deleted last time, but could you please delete?